pkg/project/auth/reviewer.go

```diff
@@ -0,0 +1,66 @@
+package auth
+
+import (
+	"fmt"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	"github.com/openshift/origin/pkg/client"
+)
+
+// Review is a list of users and groups that can access a resource
+type Review interface {
```

Comment:
Wrapper is good.  I wrote this mock before @deads2k work and missed this when integrating.